tests/bundle.test.js

```diff
@@ -0,0 +1,63 @@
+import test from "ava";
+import tokamak from "../src/tokamak.js";
+import { conclude } from "conclure";
+
+test("basic integration test", async (t) => {
+  let requireGraph = {};
+
+  const fileMap = {
+    "file:///file1.js": 'import test from "ava";\nexport default 42;',
+  };
+
+  const loader = {
+    load: function* load(url) {
+      if (!url.startsWith("file://")) {
+        throw new Error(`Don't know how to load ${url}`);
+      }
+
+      requireGraph[url] = {
+        id: url,
+        code: fileMap[url], // this gets transpiled later
+      };
+
+      return requireGraph[url];
+    },
+    isDirectory(url) {
+      return false;
+    },
+    isFile(url) {
+      return true;
+    },
+  };
+
+  const loadModule = tokamak({
+    loader,
+    logger: (level, ...messages) => {
+      console.log(`[${level.toUpperCase()}]`, ...messages);
+    },
+  });
+
+  await conclude(
+    (function* () {
+      yield loadModule("file:///file1.js");
+    })(),
```

Comment:
No need to wrap it into another generator function. Would you do something like this with promises?
```
await (async wrapper() {
   await anotherAsyncFunction(...params);
})()
```
Well, that's what you did here ;-)
You can think of `conclude` as a "top-level" `yield`